src/test/java/org/apache/commons/imaging/roundtrip/FormatInfo.java

```diff
@@ -130,4 +130,16 @@ class FormatInfo {
         this.identicalSecondWrite = identicalSecondWrite;
         this.preservesResolution = preservesResolution;
     }
+
+    @Override
+    public String toString() {
+        return "FormatInfo{" +
+            "format=" + format +
+            ", canRead=" + canRead +
+            ", canWrite=" + canWrite +
+            ", colorSupport=" + colorSupport +
+            ", identicalSecondWrite=" + identicalSecondWrite +
+            ", preservesResolution=" + preservesResolution +
+            '}';
```

Comment:
Looks harmless, but was there a reason for adding this one?

Reply:
It helped me a little when looking at some FormatInfo in the debugger of my IDE but if you don't find it useful I can remove that.

Reply:
Got it. Sounds like something that could be helpful to others too, and tests only, so really no harm in adding it. Thanks for the explanation!